bulls_and_cows/lib/api.ex

```diff
@@ -0,0 +1,34 @@
+defmodule BullsAndCows.API do
+  alias BullsAndCows
+  use GenServer
+
+  # Client
+
+  @doc """
+    Make number to guess to be picked, start the game
+  """
+  def start_link() do
+    GenServer.start_link(__MODULE__, nil)
+  end
+
+  @doc """
+    Guess the number
+  """
+  def make_turn(pid, answer) do
+    GenServer.call(pid, {:make_turn, answer})
+  end
+
+  # Server (callbacks)
+
+  @impl true
+  def init(_) do
```

Comment:
```suggestion
  def init(nil) do
```

You know it’s `nil`, so share it.